scout/adapter/mongo/variant_events.py

```diff
@@ -88,21 +88,20 @@ def unpin_variant(self, institute, case, user, link, variant):
 
         return updated_case
 
-    def order_sanger(self, institute, case, user, link, variant):
-        """Create an event for order sanger for a variant
-        and an event for order sanger for a case
+    def order_verification(self, institute, case, user, link, variant):
+        """Create an event for a variant verification for a variant
+        and an event for a variant verification for a case
 
         Arguments:
             institute (dict): A Institute object
             case (dict): Case object
             user (dict): A User object
-            link (str): The url to be used in the event
```

Comment:
Why is this removed from docstring? It is a parameter that is used...

Reply:
I have no idea!